js_modules/dagster-ui/packages/ui-core/src/assets/AssetGroupSuggest.tsx

```diff
@@ -0,0 +1,127 @@
+import {Box, Checkbox, Icon, MenuItem, Suggest} from '@dagster-io/ui-components';
+import isEqual from 'lodash/isEqual';
+import uniqBy from 'lodash/uniqBy';
+import * as React from 'react';
+
+import {AssetGroupSelector} from '../graphql/types';
+import {ClearButton} from '../ui/ClearButton';
+import {buildRepoPathForHuman} from '../workspace/buildRepoAddress';
+
+type Asset = {
+  definition: {
+    groupName: string | null;
+    repository: {name: string; location: {name: string}};
```

Comment:
Use `RepoAddress`?

Reply:
Ahh good call will fix!

Reply:
Bah, it turns out this is a partial asset coming from the backend, so we'd need to add a translation layer to make it use RepoAddress here.